# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../squashfs-tools/hotfix-sched_h.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- squashfs3.3/kernel-patches/linux-2.6.22/squashfs3.3-patch.orig	2007-11-01 06:08:56.000000000 +0100
+++ squashfs3.3/kernel-patches/linux-2.6.22/squashfs3.3-patch	2007-12-20 11:31:08.000000000 +0100
@@ -72,7 +72,7 @@
 diff -x .gitignore -Nurp linux-2.6.22/fs/squashfs/inode.c linux-2.6.22-squashfs3.3/fs/squashfs/inode.c
 --- linux-2.6.22/fs/squashfs/inode.c	1970-01-01 01:00:00.000000000 +0100
 +++ linux-2.6.22-squashfs3.3/fs/squashfs/inode.c	2007-11-01 03:09:57.000000000 +0000
-@@ -0,0 +1,2191 @@
+@@ -0,0 +1,2192 @@
 +/*
 + * Squashfs - a compressed read only filesystem for Linux
 + *
@@ -106,6 +106,7 @@
 +#include <linux/vfs.h>
 +#include <linux/vmalloc.h>
 +#include <linux/smp_lock.h>
++#include <linux/sched.h>
 +
 +#include "squashfs.h"
 +
--- squashfs3.3/kernel-patches/linux-2.6.23/squashfs3.3-patch.orig	2007-11-01 06:08:58.000000000 +0100
+++ squashfs3.3/kernel-patches/linux-2.6.23/squashfs3.3-patch	2007-12-20 11:31:17.000000000 +0100
@@ -72,7 +72,7 @@
 diff -x .gitignore -Nurp linux-2.6.23/fs/squashfs/inode.c linux-2.6.23-squashfs3.3/fs/squashfs/inode.c
 --- linux-2.6.23/fs/squashfs/inode.c	1970-01-01 01:00:00.000000000 +0100
 +++ linux-2.6.23-squashfs3.3/fs/squashfs/inode.c	2007-11-01 03:09:05.000000000 +0000
-@@ -0,0 +1,2192 @@
+@@ -0,0 +1,2193 @@
 +/*
 + * Squashfs - a compressed read only filesystem for Linux
 + *
@@ -107,6 +107,7 @@
 +#include <linux/vmalloc.h>
 +#include <linux/smp_lock.h>
 +#include <linux/exportfs.h>
++#include <linux/sched.h>
 +
 +#include "squashfs.h"
 +
--- squashfs3.3/kernel-patches/linux-2.6.24/squashfs3.3-patch.orig	2007-11-01 06:08:59.000000000 +0100
+++ squashfs3.3/kernel-patches/linux-2.6.24/squashfs3.3-patch	2007-12-20 11:31:27.000000000 +0100
@@ -72,7 +72,7 @@
 diff -x .gitignore -Nurp linux-2.6.24/fs/squashfs/inode.c linux-2.6.24-squashfs3.3/fs/squashfs/inode.c
 --- linux-2.6.24/fs/squashfs/inode.c	1970-01-01 01:00:00.000000000 +0100
 +++ linux-2.6.24-squashfs3.3/fs/squashfs/inode.c	2007-11-01 05:05:00.000000000 +0000
-@@ -0,0 +1,2192 @@
+@@ -0,0 +1,2193 @@
 +/*
 + * Squashfs - a compressed read only filesystem for Linux
 + *
@@ -107,6 +107,7 @@
 +#include <linux/vmalloc.h>
 +#include <linux/smp_lock.h>
 +#include <linux/exportfs.h>
++#include <linux/sched.h>
 +
 +#include "squashfs.h"
 +
